pom.xml

```diff
@@ -29,6 +29,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-actuator</artifactId>
 		</dependency>
+        <dependency>
```

Comment:
Indentations are off all over. Make sure your IDE is using four spaces for `.xml` and `.java` instead of tabs and fix the indentation throughout this file.